samcli/commands/build/build_context.py

```diff
@@ -169,6 +169,10 @@ def resources_to_build(self):
         result.add_layers([l for l in self._layer_provider.get_all() if l.build_method is not None])
         return result
 
+    @property
+    def is_specific_resource(self):
```

Comment:
This should get a better name, its hard to get what it means by just taking a glance. Also is this worthy of a docstring?

Reply:
Sure I will add some documentation, and for naming I could change it to `is_building_specific_resource`, what do you think?